metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -757,6 +757,12 @@ def _re_authenticate(self):
         ] = self.config.get_tableau_auth(self.site.content_url)
         self.server.auth.sign_in(tableau_auth)
 
+    @property
+    def site_content_url(self) -> Optional[str]:
+        if self.site and self.site.content_url:
+            return self.site.content_url
+        return None
```

Comment:
Seems like we always use this in a ternary. Could we just return `""` instead and avoid the ternary?

Reply:
not really - sometimes we need `/site/{self.site_content_url}`, but sometimes the prefix is `/t/` instead of site